### DISCUSSION TITLE
<title>license cost of Github Advaced Security

### DISCUSSION THREAD

**Discussion by MenakaGithub** [community/community#46827] [Enterprise]
### Select Topic Area

Product Feedback

### Body

we are using sonarqube community version for static code analysis. But now we have a plan to use Github Advanced Security for code scannig in Github enterprise edition. I'm trying to know the pricing of the " Github Advanced Security". Please can someone help me to know about the cost of GHAS license.

**Comment by ghostinhershell** [comment]
👋 Welcome to the GitHub Community on Discussions, @MenakaGithub! Happy to have you here 🙂

You can learn more about Github Advanced Security, including pricing, in our support article here: [About GitHub Advanced Security](https://docs.github.com/en/get-started/learning-about-github/about-github-advanced-security).

Check it out and let us know if you have any additional questions.
